viz/map.js

```diff
@@ -320,7 +320,7 @@ const createBikeStationLayerGroup = (year) => {
   for (let i = firstYear; i < currentYear + 1; i++) {
```

Comment:
what is `currentYear` here? Should it be year?

Reply:
updated